optimizely/utils/types.go

```diff
@@ -26,41 +26,49 @@ var intType = reflect.TypeOf(int64(0))
 
 // GetBoolValue will attempt to convert the given value to a bool
 func GetBoolValue(value interface{}) (bool, error) {
-	v := reflect.ValueOf(value)
-	if v.Type().String() == "bool" {
-		return v.Bool(), nil
+	if value != nil {
```

Comment:
Please update the unit tests to account for these scenarios.

Reply:
Test-cases implemented, please review.